core/src/main/java/net/sourceforge/jnlp/runtime/AppContextFactory.java

```diff
@@ -0,0 +1,20 @@
+package net.sourceforge.jnlp.runtime;
+
+import net.sourceforge.jnlp.runtime.classloader.DelegatingClassLoader;
+import sun.awt.SunToolkit;
+
+public class AppContextFactory {
+    public static void createNewAppContext() {
+        //set temporary classloader for EventQueue initialization
+        //already a call to AppContext.getAppContext(...) initializes the EventQueue.class
+        ClassLoader originalLoader = Thread.currentThread().getContextClassLoader();
+        DelegatingClassLoader delegatingLoader = DelegatingClassLoader.getInstance();
```

Comment:
all between line 10 and 18 should be in try { } block. Line 18 should be in finally {}

Reply:
fixed